client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -33,7 +35,22 @@ class ActivityLogDay extends Component {
 			tsEndOfSiteDay,
 			requestRestore,
 		} = this.props;
-		requestRestore( tsEndOfSiteDay );
+		requestRestore( tsEndOfSiteDay, 'day' );
+	};
+
+	trackOpenDay = () => {
+		const {
+			logs,
+			moment,
+			recordTracksEvent,
+			tsEndOfSiteDay,
+		} = this.props;
+
+		recordTracksEvent( 'calypso_activity_log_day_expand', {
+			logCount: logs.length,
+			tsEndOfSiteDay,
+			utcDate: moment.utc( tsEndOfSiteDay ).format( 'YYYY-MM-DD' ),
```

Comment:
I think you are broadly right here. The only little thing bothering me is that we are adding this `utcDate` field here precisely because it is a _rewind to a day_, but then including what could be the wrong day. Or in other words, if this field is about how far back people are rewinding, why not include it for rewinds to activities?

I'll leave it up to you, I think there are arguments either way.